configuration/parse.go

```diff
@@ -35,24 +42,53 @@ func Parse(args []string) (Configuration, error) {
 	flag.Var(&config.CveList, "exclude-vulnerability", "Comma separated list of CVEs to exclude")
 	flag.StringVar(&excludeVulnerabilityFilePath, "exclude-vulnerability-file", "./.nancy-ignore", "Path to a file containing newline separated CVEs to be excluded")
 
+	iqCommand := flag.NewFlagSet("iq", flag.ExitOnError)
+	iqCommand.StringVar(&config.User, "user", "admin", "Specify username for request")
+	iqCommand.StringVar(&config.Token, "token", "admin123", "Specify token/password for request")
+	iqCommand.StringVar(&config.Server, "server-url", "http://localhost:8070", "Specify Nexus IQ Server URL/port")
+	iqCommand.StringVar(&config.Application, "application", "", "Specify application ID for request")
+	iqCommand.StringVar(&config.Stage, "stage", "develop", "Specify stage for application")
+
 	flag.Usage = func() {
-		_, _ = fmt.Fprintf(os.Stderr, "Usage: \nnancy [options] </path/to/Gopkg.lock>\nnancy [options] </path/to/go.sum>\n\nOptions:\n")
+		_, _ = fmt.Fprintf(os.Stderr, `Usage:
+	nancy [options] </path/to/Gopkg.lock>
+	nancy [options] </path/to/go.sum>
+	nancy iq [options]
+			
+Options:
+`)
 		flag.PrintDefaults()
+		_, _ = fmt.Fprintf(os.Stderr, `
+IQ Options:
+`)
+		iqCommand.PrintDefaults()
 		os.Exit(2)
 	}
 
 	// Parse config from the command line output
-	err := flag.CommandLine.Parse(args)
-	if err != nil {
-		return config, err
+	if len(os.Args) > 1 {
+		if os.Args[1] == "iq" {
+			err := iqCommand.Parse(os.Args[2:])
+			if err != nil {
+				return config, err
+			}
+			config.IQ = true
+			config.UseStdIn = true
```

Comment:
So this might answer some of my other questions on why you did it the way you did....you are forcing the users to only use the `|` option by doing this. Personally i feel like making the api consistent so that if im using OSS or IQ server it should work the same. Whether by pipe or passing in the file to be scanned. 

Really defining IQ is just saying i want you to use a different underlying audit infrastructure. Nancy is just a facilitator and shouldn't behave differently b/c of that switch out.

Reply:
Yeah, I want to make it work for `dep` but not parsing `go.sum` (since `go list -m all` is the preferred way to do this, more authoritative)

Reply:
That's cool. Im going to open an issue to remove support for `go.sum` if we want to move away. No sense in supporting both.

Reply:
https://github.com/sonatype-nexus-community/nancy/issues/69